hud.lua

```diff
@@ -1800,18 +1800,21 @@ function courseplay.hud:setupCustomFieldEdgeButtons(vehicle,page,line)
 	vehicle.cp.hud.toggleCustomFieldEdgePathShowButton = courseplay.button:new(vehicle, page, { 'iconSprite.png', 'eye' }, 'toggleCustomFieldEdgePathShow', nil, self.buttonPosX[3], self.linesButtonPosY[line], self.buttonSize.small.w, self.buttonSize.small.h, line, nil, false);
 end
 
-function courseplay.hud:setupCombinesListPageButtons(vehicle,page)
-	for i=3, self.numLines do
-		self:addRowButton(vehicle,nil,'toggleAssignCombineToTractor', page, i, 1)
-	end;
-	local combinesListMouseWheelArea = {
-		x = self.contentMinX,
-		y = self.linesPosY[self.numLines],
-		width = self.buttonCoursesPosX[4] - self.contentMinX,
-		height = self.linesPosY[3] + self.lineHeight - self.linesPosY[self.numLines]
-	};
-	vehicle.cp.hud.combinesListMouseArea= courseplay.button:new(vehicle, 'global', nil, 'shiftCombinesList', -1, combinesListMouseWheelArea.x, combinesListMouseWheelArea.y, combinesListMouseWheelArea.width, combinesListMouseWheelArea.height, nil, -self.numLines, true, true);
-	print("vehicle.cp.hud.combinesListMouseArea: "..tostring(vehicle.cp.hud.combinesListMouseArea))
+function courseplay.hud:setupCombinesListPageButtons(vehicle,page,assignedCombinesSetting)
+	if assignedCombinesSetting then
+		for i=3, self.numLines do
+			self:addRowButton(vehicle,assignedCombinesSetting,'toggleAssignedCombine', page, i, 1)
+		end;
+		local combinesListMouseWheelArea = {
+			x = self.contentMinX,
+			y = self.linesPosY[self.numLines],
+			width = self.buttonCoursesPosX[4] - self.contentMinX,
+			height = self.linesPosY[3] + self.lineHeight - self.linesPosY[self.numLines]
+		};
+		vehicle.cp.hud.combinesListMouseArea= courseplay.button:new(vehicle, 'global', nil, 'changeListOffset', -1, combinesListMouseWheelArea.x, combinesListMouseWheelArea.y, combinesListMouseWheelArea.width, combinesListMouseWheelArea.height, nil, -self.numLines, true, true):setSetting(assignedCombinesSetting);
+	else
+		print("setup: vehicle.cp.driver.assignedCombinesSetting not found!")
```

Comment:
don't change it for this pull request, but next time include the vehicle name in the error message otherwise it isn't very useful. Also consider using courseplay:error* and courseplay:info instead of print.